static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +53,41 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically pushed to `less` command line tool if
+`less` is runnable (installed and can be executed in your shell). To be exact,
+the actual command is `less --chop-long-lines --clear-screen`.
+
+If `less` is not runnable (usually Microsoft Windows), the ouput is simply
+printed out.
+
+> Note Microsoft Windows users may also want to read
+> [this how-to](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
```

Comment:
This command's output is automatically piped to [`less`](https://en.wikipedia.org/wiki/Less_(Unix)), if available in the terminal. (The exact command used is `less --chop-long-lines --clear-screen`.) If `less` is not available (e.g. on Windows), the output is simply printed out.

> It's also possible to [enable Less paging on Windows](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).